installer/conf/omsagent.conf

```diff
@@ -84,9 +84,11 @@
   buffer_type file
   buffer_path /var/opt/microsoft/omsagent/state/out_oms_blob*.buffer
   buffer_queue_limit 10
+  buffer_queue_full_action drop_oldest_chunk
   flush_interval 20s
   retry_limit 10
   retry_wait 30s
+  max_retry_wait 9m
```

Comment:
instead of merging the later commits in the master branch, you need to rebase your private branch to the latest